ext/opentelemetry-ext-prometheus/src/opentelemetry/ext/prometheus/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "0.4.dev0"
```

Comment:
Add license info to `tests/__init__.py`.

Reply:
I believe it was @mauriciovasquezbernal who asked me to remove it, file is empty so not sure if license makes sense

Reply:
@hectorhdzg I asked to remove it from an empty __init__.py file: https://github.com/open-telemetry/opentelemetry-python/pull/378#discussion_r378236599.
I think this one should include the license.